system.exception building for iOS simulator

It would be good to add a message to the exception.

```
2021-05-07T06:43:23.0128870Z   Running: xcodebuild ONLY_ACTIVE_ARCH=YES -allowProvisioningUpdates DEVELOPMENT_TEAM= -arch x86_64 -sdk iphonesimulator -configuration Release
2021-05-07T06:43:23.0132860Z   Using working directory: /Users/runner/work/1/s/artifacts/bin/System.Xml.XmlDocument.Tests/net6.0-Release/iossimulator-x64/AppBundle/System.Xml.XmlDocument.Tests
2021-05-07T06:43:23.8772630Z /Users/runner/work/1/s/eng/testing/tests.mobile.targets(191,5): error MSB4018: The "AppleAppBuilderTask" task failed unexpectedly. [/Users/runner/work/1/s/src/libraries/System.Private.Xml/tests/XmlDocument/System.Xml.XmlDocument.Tests.csproj]
2021-05-07T06:43:23.8881610Z /Users/runner/work/1/s/eng/testing/tests.mobile.targets(191,5): error MSB4018: System.Exception: Error:  [/Users/runner/work/1/s/src/libraries/System.Private.Xml/tests/XmlDocument/System.Xml.XmlDocument.Tests.csproj]
2021-05-07T06:43:23.8927700Z /Users/runner/work/1/s/eng/testing/tests.mobile.targets(191,5): error MSB4018:  [/Users/runner/work/1/s/src/libraries/System.Private.Xml/tests/XmlDocument/System.Xml.XmlDocument.Tests.csproj]
2021-05-07T06:43:23.8955150Z /Users/runner/work/1/s/eng/testing/tests.mobile.targets(191,5): error MSB4018:    at Utils.RunProcess(String path, String args, IDictionary`2 envVars, String workingDir, Boolean ignoreErrors, Boolean silent, MessageImportance outputMessageImportance, MessageImportance debugMessageImportance) in /_/src/tasks/Common/Utils.cs:line 99 [/Users/runner/work/1/s/src/libraries/System.Private.Xml/tests/XmlDocument/System.Xml.XmlDocument.Tests.csproj]
2021-05-07T06:43:23.8962430Z /Users/runner/work/1/s/eng/testing/tests.mobile.targets(191,5): error MSB4018:    at Xcode.BuildAppBundle(String xcodePrjPath, String architecture, Boolean optimized, String devTeamProvisioning) in /_/src/tasks/AppleAppBuilder/Xcode.cs:line 368 [/Users/runner/work/1/s/src/libraries/System.Private.Xml/tests/XmlDocument/System.Xml.XmlDocument.Tests.csproj]
2021-05-07T06:43:23.9065220Z /Users/runner/work/1/s/eng/testing/tests.mobile.targets(191,5): error MSB4018:    at AppleAppBuilderTask.Execute() in /_/src/tasks/AppleAppBuilder/AppleAppBuilder.cs:line 218 [/Users/runner/work/1/s/src/libraries/System.Private.Xml/tests/XmlDocument/System.Xml.XmlDocument.Tests.csproj]
2021-05-07T06:43:23.9145800Z /Users/runner/work/1/s/eng/testing/tests.mobile.targets(191,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [/Users/runner/work/1/s/src/libraries/System.Private.Xml/tests/XmlDocument/System.Xml.XmlDocument.Tests.csproj]
2021-05-07T06:43:23.9317770Z /Users/runner/work/1/s/eng/testing/tests.mobile.targets(191,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask) [/Users/runner/work/1/s/src/libraries/System.Private.Xml/tests/XmlDocument/System.Xml.XmlDocument.Tests.csproj]
2021-05-07T06:43:23.9482320Z ##[error]eng/testing/tests.mobile.targets(191,5): error MSB4018: (NETCORE_ENGINEERING_TELEMETRY=Build) The "AppleAppBuilderTask" task failed unexpectedly.
System.Exception: Error: 

   at Utils.RunProcess(String path, String args, IDictionary`2 envVars, String workingDir, Boolean ignoreErrors, Boolean silent, MessageImportance outputMessageImportance, MessageImportance debugMessageImportance) in /_/src/tasks/Common/Utils.cs:line 99
   at Xcode.BuildAppBundle(String xcodePrjPath, String architecture, Boolean optimized, String devTeamProvisioning) in /_/src/tasks/AppleAppBuilder/Xcode.cs:line 368
   at AppleAppBuilderTask.Execute() in /_/src/tasks/AppleAppBuilder/AppleAppBuilder.cs:line 218
   at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask(ITaskExecutionHost taskExecutionHost, TaskLoggingContext taskLoggingContext, TaskHost taskHost, ItemBucket bucket, TaskExecutionMode howToExecuteTask)
2021-05-07T06:43:25.5467510Z   System.Xml.Misc.Tests -> /Users/runner/work/1/s/artifacts/bin/System.Xml.Misc.Tests/net6.0-Release/iossimulator-x64/System.Xml.Misc.Tests.dll
2021-05-07T06:43:26.0052010Z   System.Xml.Linq.SDMSample.Tests -> /Users/runner/work/1/s/artifacts/bin/System.Xml.Linq.SDMSample.Tests/net6.0-Release/iossimulator-x64/System.Xml.Linq.SDMSample.Tests.dll
```
https://dev.azure.com/dnceng/public/_build/results?buildId=1127230&view=logs&jobId=accacb66-0a14-52ef-4079-6344686d9d4c&j=accacb66-0a14-52ef-4079-6344686d9d4c&t=732ee129-7f85-548d-9cf3-22ca8690523b

--- a/src/tasks/Common/Utils.cs
+++ b/src/tasks/Common/Utils.cs
@@ -94,9 +94,9 @@ internal static class Utils
 
         if (process.ExitCode != 0)
         {
-            Logger?.LogMessage(MessageImportance.Low, $"Exit code: {process.ExitCode}");
+            Logger?.LogMessage(MessageImportance.High, $"Exit code: {process.ExitCode}");
             if (!ignoreErrors)
-                throw new Exception("Error: " + errorBuilder);
+                throw new Exception("Error: Process returned non-zero exit code: " + errorBuilder);
         }
 
         return outputBuilder.ToString().Trim('\r', '\n');

